packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -146,13 +147,19 @@ class DraggableScrollableSheet extends StatefulWidget {
   /// The default value is true.
   final bool expand;
 
+  /// A list of points that the widget should snap to. [minChildSize] and
+  /// [maxChildSize] are implicitly snapped to and should not be specified here-
+  /// pass an empty list to snap the widget between min and max.
```

Comment:
This is done for quality of life and because snapping behavior without including min and max as snap points seemed ill-defined/weird UX.